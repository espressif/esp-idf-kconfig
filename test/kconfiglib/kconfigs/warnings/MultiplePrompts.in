mainmenu "config and choice with multiple prompts"

    config A
        bool "First prompt"
        prompt "Second prompt"

    choice CHOICE_NAME
        prompt "Choice prompt"
        prompt "Choice prompt 2"

        config C1
            bool "C1"
            prompt "C1 prompt"

    endchoice
