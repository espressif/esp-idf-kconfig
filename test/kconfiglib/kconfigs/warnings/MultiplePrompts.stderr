C1 (defined at
MultiplePrompts.in:11) defined with multiple prompts in single location
warning: A (defined at
MultiplePrompts.in:3) defined with multiple prompts in single location
